connect/mirror/src/main/java/org/apache/kafka/connect/mirror/MirrorSourceConnector.java

```diff
@@ -577,6 +641,9 @@ boolean shouldReplicateTopicConfigurationProperty(String property) {
         return configPropertyFilter.shouldReplicateConfigProperty(property);
     }
 
+    boolean shouldReplicateSourceDefault(String property) {
+        return configPropertyFilter.shouldReplicateSourceDefault(property);
+    }
```

Comment:
Nit: Add an extra line after method definitions
```suggestion
    }

```